openfl-tutorials/interactive_api/medmnist_fl_3D_WIP/README.md

```diff
@@ -0,0 +1,64 @@
+# MedMNIST 2D Classification Tutorial
+
+![MedMNISTv2_overview](https://raw.githubusercontent.com/MedMNIST/MedMNIST/main/assets/medmnistv2.jpg)
```

Comment:
2D has the right title. the 3D is still WIP and would keep it separated since it's still unclear if it will be uploaded as independent tutorial or in addition to this.